src/logger.rs

```diff
@@ -0,0 +1,48 @@
+use slog::{info, o, Drain};
+use slog_async;
+use slog_term;
+
+use futures::future::FutureObj;
+
+use crate::{head::Head, Middleware, Response};
+
+/// Root logger for Tide. Wraps over logger provided by slog.SimpleLogger
+///
+/// Only used internally for now.
+pub(crate) struct RootLogger {
+    // drain: dyn slog::Drain,
```

Comment:
Should clean up this commented-out line.

Reply:
Also, if/when we expose this publicly, we should bikeshed the name a bit. (No strong opinion here)